TimeFormat.java

```diff
@@ -12,6 +12,18 @@ public static void main(String[] args) {
 		int hours = Integer.parseInt("" + args[0].charAt(0) + args[0].charAt(1));

```

Comment:
The program does not explicitly handle 00:00, which should be displayed as 12:00 AM